### PR TITLE
Add ecs-fargate-template for easy testing

### LINKED ISSUE #1: Add cloudformation file to setup a fargate cluster with one task
Add documentation to readme on how start and stop the cluster

### PR DESCRIPTION
**Pull request by fabianfett**

### CODE REVIEW

File: ec2.yml [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+Description: An example CloudFormation template for Fargate.
+Parameters:
+  
+  LatestAmiId:
+    Type: 'AWS::SSM::Parameter::Value<AWS::EC2::Image::Id>'
+    Default: '/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2'
+
+  InstanceType: 
+    Type: String
+    Default: m5.large
```
Comment (adam-fowler) [inline ec2.yml:11]: Can we use something smaller.

File: ecs-fargate.yml [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    Properties:
+      RoleName: !Join ['', [!Ref ServiceName, TaskRole]]
+      AssumeRolePolicyDocument:
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service: ecs-tasks.amazonaws.com
+          Action: 'sts:AssumeRole'
+      Policies:
+      - PolicyName: s3access
```
Comment (adam-fowler) [inline ecs-fargate.yml]: Can we prefix names with aws-vapor-test so we know where they are coming from. Also will mean less name clashes

File: ec2.yml [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+        - Effect: Allow
+          Principal:
+            Service: ec2.amazonaws.com
+          Action: 'sts:AssumeRole'
+      Policies:
+      - PolicyName: s3access
```
Comment (adam-fowler) [inline ec2.yml]: Can we prefix names with aws-vapor-test so we know where they are coming from. Also will mean less name clashes

File: ec2.yml (hunk 1 above)
Comment (fabianfett) [reply ec2.yml:11]: Nah. To quote Cersei Lannister:
https://www.youtube.com/watch?v=ab6GyR_5N6c&feature=youtu.be&t=92

**Review by adam-fowler — APPROVED** [review]

**Pull request merged**